src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -68,7 +68,8 @@ public PersistentDataStore() {
         String userName = (String) entity.getProperty("username");
         String passwordHash = (String) entity.getProperty("password_hash");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        User user = new User(uuid, userName, passwordHash, creationTime);
+        String bio = (String) entity.getProperty("bio");
```

Comment:
One thing to think about: how does this affect old data? (Users that don't have bios)

The workaround for this can be to require everybody to delete their old data, but in that case just let the team know that they need to delete their old data.

Reply:
I believe we're just going to have to delete old data since we only made a few accounts.